<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetControl</class>
 <widget class="QWidget" name="WidgetControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>434</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>434</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>434</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>60</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>60</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <property name="spacing">
    <number>20</number>
   </property>
   <item row="1" column="0">
    <widget class="QWidget" name="widget_4" native="true"/>
   </item>
   <item row="0" column="2">
    <widget class="QWidget" name="widget_3" native="true"/>
   </item>
   <item row="1" column="2">
    <widget class="QWidget" name="widget_6" native="true"/>
   </item>
   <item row="0" column="0">
    <widget class="QWidget" name="widget_1" native="true"/>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="widget_2" native="true"/>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="widget_5" native="true"/>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="button_start">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border-image: url(:/images/button/button1.png);
	color: rgb(66, 66, 66);
	font: 24px &quot;.Helvetica Neue DeskInterface&quot;;
}
QPushButton:pressed{
border-image: url(:/images/button/button1_press.png);
}
QPushButton:disabled{
border-image: url(:/images/button/button1_press.png);
}</string>
     </property>
     <property name="text">
      <string>开始检测</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
